app/javascript/react/pages/Home/Home.js

```diff
@@ -19,7 +35,14 @@ const Home = () => {
   const addInputElementHandler = (e) => {
     e.preventDefault();
     if (nameInput || valueInput) {
```

Comment:
why is this needed?

Reply:
it was initially needed when button wasn't disabled, not needed now